src/charm.py

```diff
@@ -30,7 +30,8 @@ def _validated_address(address: str) -> str:
 
     parsed = urlparse(address)
     if not parsed.netloc or any([parsed.scheme, parsed.path, parsed.params, parsed.query]):
-        logger.error("Invalid address (should only include netloc): %s", address)
+        logger.error("Invalid address : %s", address)
+        logger.error("Valid address must be in host:port format")
```

Comment:
Splitting into two entries is neater, but these may end up being separated by other incoming logs so maybe one line is better after all?

Reply:
On an modern ANSI compliant terminal (nearly all) the error messages will be visible in bold red. If one can be spotted easily so will the other. Besides `juju debug-log --replay | grep ERROR` will show them together.